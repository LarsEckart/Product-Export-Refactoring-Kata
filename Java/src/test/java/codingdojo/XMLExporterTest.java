package codingdojo;

import org.junit.Before;
import org.junit.Test;
import org.approvaltests.Approvals;

import java.text.ParseException;
import java.util.ArrayList;

public class XMLExporterTest {


    private ArrayList<Order> orders;

    @Test
    public void testSomething() throws Exception {

        // TODO: Finish this test case
    }


}
