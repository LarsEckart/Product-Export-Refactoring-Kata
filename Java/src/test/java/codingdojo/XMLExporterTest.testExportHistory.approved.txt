<?xml version="1.0" encoding="UTF-8"?>
<orderHistory>
<order date='2018-09-01T13:24Z' totalDollars='195.95'>
<product id='LIPSTICK01'>Cherry Bloom</product>
<product id='LIPSTICK01'>Cherry Bloom</product>
<product id='PURFUME01'>Wild Rose</product>
<product id='SKIN_CREAM01'>Cocoa Butter</product>
<product id='EVENT01'>Eyeshadow Masterclass</product>
</order>
<order date='2017-08-31T14:02Z' totalDollars='229.96'>
<product id='TOOL01'>Eyelash curler</product>
<product id='TOOL01'>Blusher Brush</product>
<product id='PURFUME01'>Wild Rose</product>
<product id='EVENT02'>Makeover</product>
</order>
</orderHistory>
